numpy/random/tests/test_seed_sequence.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from numpy.testing import assert_array_equal
 
-from numpy.random.bit_generator import SeedSequence
```

Comment:
Any reason to not import from numpy.random?  (I haven't checked, but think SS is exposed).

Reply:
adopted